elements/campaign-display/components/campaign-display-image.js

```diff
@@ -3,13 +3,14 @@ import React, { PropTypes } from 'react';
 export default function CampaignDisplayImage (props) {
   return (
     <div className='campaign-display-image'>
-      <a href={props.url}>
-        <img src={props.image} />
+      <a href={props.clickthrough_url}>
```

Comment:
which attributes?

So we couldn't get away with putting the attributes directly on the `<campagin-display>` as it is?

I think that would be ideal.